README.md

```diff
@@ -91,17 +91,21 @@ When generating a new personal access token, the scope only needs to
 encompass `public_repo` (or `repo` if you're accessing a private repo).
 
 If you'd like to download Red Hat-only internal images from S3, you'll
-need to create a key file in `~/.config/cockpit-dev/s3-keys/[endpoint]`.
-The contents of this file should be a single line containing the "access
-key" and the "secret key" separated by whitespace.
+need to create a key file in `~/.config/cockpit-dev/s3-keys/[domain]`.
+The `[domain]` can be any non-toplevel domain which contains the S3 URL
+in question.  The contents of this file should be a single line
+containing the "access key" and the "secret key" separated by
+whitespace.
 
-For the currently configured mirrors this means that you'd expect to
-have the following files:
+For the currently configured mirrors this means that you could have any
+of the following files:
 
+- `~/.config/cockpit-dev/s3-keys/cockpit-images.eu-central-1.linodeobjects.com`
 - `~/.config/cockpit-dev/s3-keys/eu-central-1.linodeobjects.com`
-- `~/.config/cockpit-dev/s3-keys/us-east-1.linodeobjects.com`
+- (either of the above, with `us-east` instead of `eu-central`); or
+- `~/.config/cockpit-dev/s3-keys/linodeobjects.com`
```

Comment:
This is really nice! Is there any reason to not make this the recommended variant? I.e. put it first perhaps?

Reply:
It would make the ordering weird with respect to the "either of the above".  I'll figure out a way to rewrite it.

Reply:
I meant, would you actually recommend that "we" (humans + CI) use this  particular name? It seems intuitive to me to name it by provider, as the credentials are valid for all regions.

Reply:
It's not always true.  Our upload credentials are tied to the region, for example.